source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
```

Comment:
Totally @melissa. If the docs are stale elsewhere, we should address it. I'm raising the flag a little higher here as these includes are re-used in several places across the docs, so these changes will ripple, and I want to make sure those pages are also consistent.